packages/api/src/controllers/taskController.js

```diff
@@ -619,4 +622,20 @@ async function notifyAssignee(userId, taskId, taskTranslationKey, farmId) {
   );
 }
 
+/**
+ * Checks if the current user can complete the task
```

Comment:
JSDoc comments need a period at the end of this first description sentence. (I know that's very picky, but it's meaningful to the software that processes these, should we ever use it.)